media/css/mozorg/technology.scss

```diff
@@ -878,11 +807,11 @@ html[dir="rtl"] #technologies > section {
         }
 
         p {
-            @include font-size(16px);
+            @include font-size-level5;
```

Comment:
Yeah, I'm gonna make this 16px. The best long-term solution will be to redesign the page.